pie-website/src/components/Footer.js

```diff
@@ -28,7 +28,6 @@ function Footer() {
             <Link to="/">Pioneers in Engineering</Link>
           </h2>
           <div>
-            <h3>Get in Touch</h3>
```

Comment:
Decided to remove this header because I think it's obvious that the information below are methods to communicate with us

Reply:
Plus, it took up vertical real estate that i need to eat up with my new blurb